dataland-frontend/src/utils/FileUploadUtils.ts

```diff
@@ -29,8 +30,16 @@ export async function uploadFiles(
 ): Promise<void> {
   const documentControllerApi = await new ApiClientProvider(getKeycloakPromise()).getDocumentControllerApi();
   for (const fileToUpload of files) {
-    const fileIsAlreadyInStorage = (await documentControllerApi.checkDocument(fileToUpload.reference)).data
-      .documentExists;
+    let fileIsAlreadyInStorage: Boolean;
+    try {
+      await documentControllerApi.checkDocument(fileToUpload.reference)
+      fileIsAlreadyInStorage = true;
+    }  catch (error) {
+      if (error instanceof AxiosError && (error as AxiosError).response!.status != 404) {
```

Comment:
== 404, oder?   Weil das ist ja der erwartete Error wenn es die documentId nicht gibt.

Reply:
Aaah, doof. Ich wollte nur in dem Fall den error throwen, hab mich dann strukturell aber umentschieden. To be pushed.

Reply:
👍